### PR TITLE
Update readme

### LINKED ISSUE #167: Add badge "chat on slack"
image

### PR DESCRIPTION
**Pull request by bowni**
Fixes #167 #168 #175

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -140,6 +144,12 @@ Example: `n_repeats: 2`
 `dataset_name`: `'mnist'` (default) or `'cifar10'`  
 MNIST and CIFAR10 are currently supported. They come with their associated modules in `/datasets` for loading data, pre-processing inputs, and define a model architecture.
 
+**Note on validation and test datasets**:  
+- the datasets module must provide separated train and test sets
+- the train set is then further split into separated train and validation sets
+- in the multi-partner learning computations, this global validation set is used for early stopping and the global test set is used for performance evaluation
+- additionally, after the train data samples are split amongst partners according to the scenario configuration, a partner's local train set is split into separated train, validation and test sets. Currently, the local validation and test set are not used, but they are available for further developments of multi-partner learning and contributivity measurement approaches
```
Comment (natct10) [inline README.md:151]: a partner's local train set is "splitted", nah?

Comment (RomainGoussault) [reply README.md:151]: - The dataset modules must provide separated train and test sets (named global train set and global test set).
- This global train set is then further split into a global train set and a global validation set.
- In the multi-partner learning computations, this global validation set is used for early stopping and the global test set is used for performance evaluation.
- The global train set is split amongst partner (according to the scenario configuration) to populate the local datasets.
- For each partner, the local dataset is split into separated train, validation and test sets. Currently, the local validation and test set are not used, but they are available for further developments of multi-partner learning and contributivity measurement approaches.

File: README.md [hunk 2]
```diff
@@ -115,9 +116,12 @@ Finally, with given scenarios and multi-partner learning approaches, we can addr
 
    Under `scenario_params_list`, enter a list of sets of scenario(s). Each set starts with ` - dataset_name:` and must have only one `partners_count` value. The length of `amount_per_partners`, `corrupted_datasets` (and `samples_split_option` when the advanced definition is used) must match the `partner_counts` value. If for a given parameter multiple values are specified, e.g. like for `agregation_weighting` in the first scenario set of the above example, all possible combinations of parameters will be assembled as separate scenarios and run.
    
-3. Then execute `main.py -f config.yml`. Add the `-v` argument if you want a more verbose output. 
+1. Then execute `main.py -f config.yml`. Add the `-v` argument if you want a more verbose output. 
 
-4. A `results.csv` file will be generated in a new folder for your experiment under `/experiments/<your_experiment>`. You can read this raw `results.csv` file or use the notebooks in `/notebooks`.
+1. A `results.csv` file will be generated in a new folder for your experiment under `/experiments/<your_experiment>`. You can read this raw `results.csv` file or use the notebooks in `/notebooks`.
```
[the remark this answers is not in the document]
Comment (natct10) [reply README.md:121]: Nah it will be auto incremented in the md render!! Nice feature discovered from @bowni

File: README.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 [![Build Status](https://travis-ci.org/SubstraFoundation/distributed-learning-contributivity.svg?branch=master)](https://travis-ci.org/SubstraFoundation/distributed-learning-contributivity)
 [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/SubstraFoundation/distributed-learning-contributivity/blob/master/run_experiment_on_google_collab.ipynb)
+[![Discuss on Slack](https://img.shields.io/badge/chat-on%20slack-orange)](https://substra.us18.list-manage.com/track/click?e=2effed55c9&id=fa49875322&u=385fa3f9736ea94a1fcca969f)
```
[the remark this answers is not in the document]
Comment (bowni) [reply README.md:3]: Do you think it is even possible? cc @ClementMayer

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (bowni) [reply README.md:121]: This is actually core markdown :)

File: README.md (hunk 1 above)
Comment (bowni) [reply README.md:151]: - @natct10 https://fr.bab.la/conjugaison/anglais/split 😄 
- @RomainGoussault OK

**Comment by bowni** [comment]
> I'd have preferred a blue badge but orange is fine!

I tested it first, but it was quite weird to have one green and two blues. I thought that 3 different colors looked better

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (bowni) [reply README.md:3]: I checked the documentation: you can't have an invitation link pointing to a specific channel.

**Comment by bowni** [comment]
All fixed - @RomainGoussault can you validate and merge please?

File: README.md (hunk 1 above)
Comment (natct10) [reply README.md]: >     * @natct10 https://fr.bab.la/conjugaison/anglais/split smile
> 
>     * @RomainGoussault OK

Ok noted, thx for the feedback!
